src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -596,12 +587,17 @@ std::shared_ptr<GLHEVertSingle> GetSingleBH(EnergyPlusData &state, std::string c
 std::shared_ptr<GLHEVertArray> GetVertArray(EnergyPlusData &state, std::string const &objectName)
 {
     // Check if this instance of this model has already been retrieved
-    for (auto &thisProp : state.dataGroundHeatExchanger->vertArraysVector) {
-        // Check if the type and name match
-        if (objectName == thisProp->name) {
-            return thisProp;
-        }
-    }
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:601]:(style),[useStlAlgorithm],Consider using std::find_if algorithm instead of a raw loop.